test/unit/rules/no-pointer-down-event-binding-test.js

```diff
@@ -1,61 +1,61 @@
 'use strict';
 
-const { ERROR_MESSAGE } = require('../../../lib/rules/no-down-event-binding');
+const { ERROR_MESSAGE } = require('../../../lib/rules/no-pointer-down-event-binding');
 const generateRuleTests = require('../../helpers/rule-test-harness');
 
 generateRuleTests({
-  name: 'no-down-event-binding',
+  name: 'no-pointer-down-event-binding',
 
   config: true,
 
   good: [
```

Comment:
Personally, I would keep the old good cases as they are still allowed and to make sure not to introduce any regressions.

Reply:
I think if this pointer-down linting concern had been introduced first, we wouldn't have had a reason to include `key` events in the scope of the tests, so it made sense to me to remove those. I don't think anything is really gained by having those included. I can add those back in the "good" checks if you feel strongly on that point, though.

Reply:
Nope, no strong feelings on this one. I'm just a contributor btw, so not my decision to make. But I've seen this mentioned before in PRs for this project, that's why I mentioned it.